ntfs_fr.md

```diff
@@ -356,8 +356,10 @@ Colonne | Type | Contrainte | Commentaire
 --- | --- | --- | ---
 stop_time_id | chaine | Optionnel | Identifiant unique de l'horaire dans le jeu de données. Cette information n'est pas pérenne et permet uniquement de faire le lien entre un horaire (fichier [`stop_times.txt`](#stop_timestxt-requis)) et un commentaire (fichier [`comments.txt`](#commentstxt-optionnel)) en utilisant le fichier [`comment_links.txt`](#comment_linkstxt-optionnel). Si ce champ n'est pas fourni, l'horaire ne pourra pas êter lié à un commentaire.
 trip_id | chaine | Requis | Identifiant de la circulation
-arrival_time | heure | Requis | Heure d'arrivée. Si l'heure d'arrivée n'est pas connue, elle doit être estimée par le système fournissant les données et le champ *stop_time_precision* doit être spécifié à 1. Si la descente est interdite à cet arrêt, l'heure d'arrivée doit être indiquée et le champ *drop_off_type* doit être spécifié à 1.
-departure_time | heure | Requis | Heure de départ. Si l'heure de départ n'est pas connue, elle doit être estimée par le système fournissant les données et le champ *stop_time_precision* doit être spécifié à 1. Si la montée est interdite à cet arrêt, l'heure de départ doit être indiquée et le champ *pickup_type* doit être spécifié à 1.
+arrival_time | heure | Optionnel | Heure d'arrivée. Si l'heure d'arrivée n'est pas connue, elle doit être estimée par le système fournissant les données et le champ *stop_time_precision* doit être spécifié à 1. Si la descente est interdite à cet arrêt, l'heure d'arrivée doit être indiquée et le champ *drop_off_type* doit être spécifié à 1. La valeur doit être obligatoirement nulle si *start_pickup_drop_off_window* ou *end_pickup_drop_off_window* est défini.
+departure_time | heure | Optionnel | Heure de départ. Si l'heure de départ n'est pas connue, elle doit être estimée par le système fournissant les données et le champ *stop_time_precision* doit être spécifié à 1. Si la montée est interdite à cet arrêt, l'heure de départ doit être indiquée et le champ *pickup_type* doit être spécifié à 1. La valeur doit être obligatoirement nulle si *start_pickup_drop_off_window* ou *end_pickup_drop_off_window* est défini.
+start_pickup_drop_off_window | heure | Optionnel | Heure d'ouverture du service de TAD sur le point d'arrêt. Valeur obligatoire si *end_pickup_drop_off_window* est défini, nulle si *arrival_time* ou *departure_time* est défini.
+end_pickup_drop_off_window | heure | Optionnel | Heure de fermeture du service de TAD sur le point d'arrêt. Valeur obligatoire si *start_pickup_drop_off_window* est défini, nulle si *arrival_time* ou *departure_time* est défini.
```

Comment:
"Valeur obligatoire si *start_pickup_drop_off_window* est défini, nulle si *arrival_time* ou *departure_time* est défini."
Préférer
"Valeur obligatoire si *start_pickup_drop_off_window* est défini, vide si *arrival_time* ou *departure_time* est défini."

Reply:
Ok in https://github.com/hove-io/ntfs-specification/pull/51/commits/afaf4302c573f65ba9e3251ff253da0e1ff268c4